ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -48,6 +52,15 @@ def _get_group(self, base_url, group_name):
         except Exception, e:
             raise e
 
+    def _get_organization(self, base_url, org_name):
+        url = base_url + self._get_action_api_offset() + '/organization_show?id=' + org_name
+        try:
+            content = self._get_content(url)
+            content_dict = json.loads(content)
+            return content_dict['result']
+        except Exception, e:
```

Comment:
There are also lots of programming errors that you can make, that cause different exceptions, which are nothing to do with the server's response. Not swallowing those exceptions helps fix the code. Hence why we have a CKAN policy of making the effort to identify the correct exception in except clauses. It's also mentioned in PEP8:

> When catching exceptions, mention specific exceptions whenever possible instead of using a bare except: clause.